src/CoreBundle/private/library/classes/TCMSTableEditor/TCMSTableEditorMenuItem.class.php

```diff
@@ -66,13 +66,13 @@ class TCMSTableEditorMenuItem
 
     /**
      * bootstrap button style (default: btn-primary)
-     * possible values in bootstrap 3.0: btn-default, btn-primary, btn-success, btn-info, btn-warning, btn-danger, btn-link.
+     * possible values in bootstrap 4.1: btn-secondary, btn-primary, btn-success, btn-info, btn-warning, btn-danger, btn-link.
      *
      * @see http://getbootstrap.com/css/#buttons
      *
      * @var string
      */
-    protected $sButtonStyleClass = 'btn-default';
+    protected $sButtonStyleClass = 'btn-secondary';
```

Comment:
I'd expect btn-primary to be some kind of default (because why have something secondary if there isn't something primary?). Why is secondary better here (and other places in this PR)?

Reply:
Nope... secondary IS default. Blame Bootstrap. The style of btn-default is now the style of btn-secondary. We can not change this here without changing all buttons to a very primary color. Nope nope.